### DISCUSSION TITLE
Should `display` be `grid` for the library to take effect?

### DISCUSSION THREAD

**Discussion by fujianjin6471** [simonwep/viselect#154] [General]
In my case, I want to use this library for a list (one column), but when I remove `display: grid` from `.container` in CSS,  it doesn't work. Does this library work when `display` is not `grid`?

Thanks in advance!

**Comment by simonwep** [comment]
Could you provide a reproduction? Even the demo-page has two sections where `display: flex` is used, it should work with any kind of container.
